### ISSUE TITLE
[api-extractor] the dtsRollup does not correctly emit reexported types from an external package

### ISSUE THREAD

**Issue by pgonzal** [Microsoft/web-build-tools#907]
Suppose **index.ts** does this:

**api-extractor-test-02/src/index.ts**
```ts
export { ReexportedClass } from 'api-extractor-test-01';
```

The emitted rollup says "import" instead of "export" looks like this:

**api-extractor-test-02/dist/internal/api-extractor-test-02.d.ts**
```ts
import { ReexportedClass } from 'api-extractor-test-01';
```

Generally it's not a great software design to be reexporting aliased types from another package.  But API Extractor should support it.

**Comment by pgonzal** [comment]
@patmill

**Comment by pgonzal** [comment]
This looks good.  Sorry it took me a few days to reply.  I wanted to test it in a large repo first to make sure it doesn't cause any unforeseen regressions.

**Comment by pgonzal** [comment]
This fix was published as **api-extractor** beta version 7.0.2.
